### PR TITLE
Feature/svitlykdm/develop

### PR DESCRIPTION
**Pull request by DmytroSvitlyk**
completed task2 by using Java language for client and C for server.

### CODE REVIEW

File: tic-tac-toe-client/src/main/java/Constants.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+public interface Constants {
```
Comment (andrii-s-shpitser-lpnu-ua) [inline tic-tac-toe-client/src/main/java/Constants.java:1]: I see here code (which is for task#2) and tests (which is for task#3)
So
1) what task is it for?
2) if it is for task#2 then what code will be run on HW?

Comment (DmytroSvitlyk) [reply tic-tac-toe-client/src/main/java/Constants.java:1]: 1. There are tasks 2 and 3 in latest 4 commits in this branch.
2. main.c file in folder "tic-tac-toe-server" is code to be run on HW

Comment (andrii-s-shpitser-lpnu-ua) [reply tic-tac-toe-client/src/main/java/Constants.java:1]: in futer please create separate PR for seprate tasks

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

File: tic-tac-toe-client/src/main/java/Constants.java (hunk 1 above)
Comment (DmytroSvitlyk) [reply tic-tac-toe-client/src/main/java/Constants.java:1]: Okay, I got it. Sorry

**Pull request merged**